masterplan_tools/specification/specification_validator.py

```diff
@@ -0,0 +1,27 @@
+"""
+The module is aimed to validate input json data according to various schemas.
+"""
+from pathlib import Path
+import json
+import os
+from jsonschema import validate, RefResolver, Draft4Validator
+
+package_directory = os.path.dirname(os.path.abspath(__file__))
+
+
+class SpecificationValidator:
+    """
+    todo
```

Comment:
Not very informative. Also, some linters only check `TODO:` or `todo:` (with a colon) as a token for TODOs